operator/netchecks_operator/main.py

```diff
@@ -693,6 +694,7 @@ def create_job_spec(
         image_pull_policy=settings.probe.image.pullPolicy,
         command=command,
         volume_mounts=volume_mounts,
+        resources=client.V1ResourceRequirements(limits=settings.probe.resources["limits"],requests=settings.probe.resources["requests"]),
```

Comment:
This needs to work for the case where there are no resources defined.

Probably cleaner to create the `V1ResourceRequirements` instance above, then pass in to the `V1Container`